inst/stan/wwinference.stan

```diff
@@ -167,6 +168,7 @@ parameters {
   real log_scaling_factor;
   matrix[n_subpops-1,n_weeks] non_cent_spatial_dev_ns_mat;
   vector[n_weeks] norm_vec_aux_site;
+  cholesky_factor_corr[n_subpops-1] L_Omega;
```

Comment:
This should be defined only if we're in the "inferred correlation but no distance matrix case". The way to do that in stan is to make it size zero in other cases.

See https://discourse.mc-stan.org/t/if-else-statement-inside-parameter-block/13937/2 for details